scip.proto

```diff
@@ -102,8 +103,37 @@ message Document {
   // can be used for other purposes as well, for example testing or when working
   // with virtual/in-memory documents.
   string text = 5;
+
+  // Specifies the encoding used for source ranges in this Document.
+  PositionEncoding position_encoding = 6;
```

Comment:
I'm open to adding a comment with recommendations. Something like "If you are unsure which encoding to use, we recommend `UTF16CodeUnitOffsetFromLineStart` if you are working with JS/JVM and `UTF8CodeUnitOffsetFromLineStart` if you working with Python". Or at least something about how to approach deciding which one to use. Not a blocking comment.

Reply:
Added a recommendation.